example-notebooks/WFS_workflow_example.ipynb

```diff
@@ -1,5 +1,34 @@
 {
  "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "92254022",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "---\n",
+    "title: Get Fire Perimeters from Web Feature Service\n",
+    "description: An exmaple that shows how to explore data available through a web feature service, and how to filter data temporally, spatially, and by property.\n",
+    "author: Tempest McCabe\n",
+    "date: March, 8, 2023\n",
+    "execute:\n",
+    "  cache: true\n",
+    "  freeze: true\n",
+    "---"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "06c64d14",
+   "metadata": {},
+   "source": [
+    "You can launch this notebook using mybinder by clicking the button below. \n",
+    "\n",
+    "[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/mccabete/veda-docs.git/url-update?labpath=%2Fexample-notebooks%2FWFS_workflow_example.ipynb)"
```

Comment:
I'm just going to make this one change to the url to make it point to the main branch of the main repo after this merges

```suggestion
    "[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/NASA-IMPACT/veda-docs/HEAD?labpath=example-notebooks%2FWFS_workflow_example.ipynb)"
```

Reply:
Done! :)